src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,61 @@
+.calendar {
+  max-width: 7 * $cell-size + 7 * $cell-gap + 10px;
```

Comment:
There is no point in exactly max-width, the block will never be less than or more than 7 columns wide and 6 pixels between them. And the equation looks complicated. Maybe it can be simplified

Reply:
You can, but it's ok